domain/model/state/state.go

```diff
@@ -793,71 +788,178 @@ func (s *State) ListAllModels(ctx context.Context) ([]coremodel.Model, error) {
 		return nil, errors.Trace(err)
 	}
 
-	modelStmt := `
-SELECT uuid,
-       name,
-       cloud_name,
-       cloud_region_name,
-	   model_type,
-	   owner_uuid,
-	   owner_name,
-	   cloud_credential_name,
-	   cloud_credential_owner_name,
-	   cloud_credential_cloud_name,
-	   life
+	query := `
+SELECT &dbModel.*
 FROM v_model
 `
 
-	rval := []coremodel.Model{}
-	err = db.StdTxn(ctx, func(ctx context.Context, tx *sql.Tx) error {
```

Comment:
Thanks for replacing these, this and model config are the last domains to extensivly use StdTxn so it'll be great once they're converted